pktgen/app/pktgen-tool.c

```diff
@@ -291,6 +291,9 @@ int parse_user_config(void) {
 
   if (config_lookup_int(&cfg, "enable_gtpu", &val)) {
     enable_gtpu = val;
+    /* register user additional user signals in case pktgen needs to handled via signals */
+    signal(SIGUSR1, start_gtp_traffic);
+    signal(SIGUSR2, quit_gtp_traffic);
```

Comment:
This is a nice thought. Please also document this way of controlling the traffic_gen in README with an example

Reply:
Thanks for the reminder. I forgot to document this.